index.js

```diff
@@ -2023,16 +2043,18 @@ if (!window.console) {
    * //A connect with a nonstandard timeout
    * var cb = ClearBlade.Messaging({"timeout":15}, callback);
    */
-  ClearBlade.prototype.Messaging = function(options, callback) {
+  var messageCallbacks = {};
```

Comment:
should we store these on the messaging object so they are monitored on a messaging instance basis so they're not shared across messaging instances on the same clearblade instance?

Reply:
Yeah, unfortunately it causes the messaging to stop working -- tried it again just now